tbx/project_styleguide/templates/patterns/pages/blog/blog_detail.html

```diff
@@ -23,7 +23,7 @@
 {% endblock %}
 
 {% block content %}
-    <div class="grid grid--spacer-large streamfield">
+    <div class="grid grid--spacer-large streamfield lg:relative">
```

Comment:
I don't think the relative and absolute positioning is changing the avatar position - only the top margin is having and effect.